qt/scientific_interfaces/Indirect/JumpFit.cpp

```diff
@@ -42,9 +42,10 @@ void JumpFit::setup() {
   m_jfTree->addProperty(m_properties["StartX"]);
   m_jfTree->addProperty(m_properties["EndX"]);
 
-  // Fitting function
-  m_properties["FitFunction"] = m_grpManager->addProperty("Fitting Parameters");
-  m_jfTree->addProperty(m_properties["FitFunction"]);
```

Comment:
I assume these are all fitting functions

Reply:
Yes, these are currently all the available fitting functions for JumpFit.